services/fastmail/config.json

```diff
@@ -19,16 +27,114 @@
       "prio": 20
     },
     {
+      "name": "*",
+      "type": "MX",
+      "content": "in1-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 10
+    },
+    {
+      "name": "*",
+      "type": "MX",
+      "content": "in2-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 20
+    },
+    {
+      "name": "mail",
+      "type": "MX",
+      "content": "in1-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 10
+    },
+    {
+      "name": "mail",
+      "type": "MX",
+      "content": "in2-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 20
+    },    {
       "name": "mail",
       "type": "CNAME",
-      "content": "www.fastmail.fm",
+      "content": "mail.fastmail.com",
+      "ttl": 3600
+    },
+    {
+      "name": "mesmtp._domainkey",
+      "type": "TXT",
+      "content": "{{dkim}}",
+      "ttl": 3600
+    },
+    {
+      "name": "_adsp._domainkey",
+      "type": "TXT",
+      "content": "dkim=unknown",
       "ttl": 3600
     },
     {
-      "type": "SPF",
+      "type": "TXT",
```

Comment:
Although the `SPF` record type is deprecated, it is fine to keep it here, as our system serves a `TXT` record when an `SPF` record is added, so we shouldn't change this line.